julia-mode.el

```diff
@@ -67,6 +67,35 @@ User can still use `abbrev-mode' or `expand-abbrev' to substitute
 unicode for LaTeX even if disabled."
   :type 'boolean)
 
+(defconst julia-mode--latexsubs-partials
+  (let ((table (make-hash-table :test 'equal)))
+    (maphash (lambda (latex subst)
+               (cl-assert (string= (substring latex 0 1) "\\") nil
+                          "LaTeX substitution does not start with \\.")
+               (let ((len (length latex)))
+                 (cl-assert (< 1 len) nil "Trivially short LaTeX subtitution")
+                 (cl-loop for i from 2 to len
+                          do (puthash (substring latex 1 i) t table))))
+             julia-mode-latexsubs)
+    table)
+  "A hash table containing all partial strings from the LaTeX abbreviations in `julia-mode-latexsubs' as keys. Values are always `t', the purpose is to represent a set.")
```

Comment:
Docstrings shouldn't be wider than 80 characters. Byte compiler complains.

https://www.gnu.org/software/emacs/manual/html_node/elisp/Documentation-Tips.html